### DISCUSSION TITLE
iOS 快捷指令带标签版本

### DISCUSSION THREAD

**Discussion by o1xhack** [flomoapp/3rd-party-tools#13] [分享创造]
https://mp.weixin.qq.com/s?__biz=MzI0NDAxNjA5OA==&mid=2657963423&idx=2&sn=a825ffddd2bf7c847b55dcadc4d273fa&chksm=f2fddb12c58a5204345698f44878330fc83c2ab340cd5734ba395007201b955a7c8b98057dd4&token=1814730313&lang=zh_CN#rd

文章推荐 flomo 以及自制的更新版快捷指令，支持加入标签，选择~

**Comment by ClisMakise** [comment]
分享相册照片到此会显示「不支持的URL」，请问是否可以解决？

**Reply by o1xhack** [reply]
我有点不确定是否可以支持上传图片。我试一下
